src/routes/threlte/PassiveSketch.svelte

```diff
@@ -278,31 +305,31 @@
 		{/each}
 
 		{#each $previewGeometry as geom (geom.uuid)}
-			{#if geom.type === 'line'}
+			{#if isGeomType(geom, "line")}
 				<Line
 					start={geom.start}
 					end={geom.end}
-					id={null}
+					id={geom.uuid}
```

Comment:
are these correct? setting the id to null just felt _**wrong**_!

Reply:
I honestly can't remember why I would have set these to null! no idea

Reply:
Sweet! uuid it is then!